File: src/main/java/com/example/appforauslenderamt/entity/PlaceOfResidence.java
package com.example.appforauslenderamt.entity;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@Builder
@Getter
@AllArgsConstructor
public class PlaceOfResidence {

    @JsonProperty("postal_code")
    private final String postalCode;
    @JsonProperty("place")
    private final String place;
    @JsonProperty("street")
    private final String street;
    @JsonProperty("house_number")
    private final Integer houseNumber;

    @Override
    public String toString() {
        return postalCode + ", " + place + ", " + street + ", " + houseNumber;
    }
}

File: Frontend/src/app/Models/BackendModels/PlaceOfResidence.ts

export class PlaceOfResidence {

    postal_code: string
    place: string
    street: string
    house_number: string
    constructor(
        postalCode: string,
        place: string,
        street: string,
        houseNumber: string
    ) {
        this.postal_code = postalCode
        this.place = place
        this.street = street
        this.house_number = houseNumber
    }


}
